unity-renderer/Assets/Scripts/MainScripts/DCL/DataStore/DataStore_SceneBoundariesChecker_Extensions.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using DCL;
+using DCL.Controllers;
+using DCL.Models;
+using UnityEngine;
+
+public static class DataStore_SceneBoundariesChecker_Extensions 
+{
+    public static void Add(this DataStore_SceneBoundariesChecker self,IDCLEntity entity, IOutOfSceneBoundariesHandler handler)
+    {
+        if (self.componentsCheckSceneBoundaries.ContainsKey(entity))
+            self.componentsCheckSceneBoundaries[entity].Add(handler);
```

Comment:
you could use `TryGetValue` here to avoid the dictionary to look for the key twice